kubectl-cloudflow/docker/client.go

```diff
@@ -128,23 +147,16 @@ func imageMatchesNameAndTag(image types.ImageSummary, imageNameAndTag string) bo
 	return false
 }
 
-// GetCloudflowStreamletDescriptorsForImage extracts the configuration of a Cloudflow label from a docker image
+// GetStreamletDescriptorsForImage extracts the configuration of a Cloudflow label from a docker image
 // and the image with digest in a struct
-func GetCloudflowStreamletDescriptorsForImage(cli *client.Client, imageName string) (cloudflowapplication.CloudflowStreamletDescriptorsDigestPair, string) {
-
-	_, pullError := PullImage(cli, imageName)
-	if pullError != nil {
-		util.LogAndExit("Failed to pull image %s: %s", imageName, pullError.Error())
-	}
-
+func GetStreamletDescriptorsForImage(cli *client.Client, imageName string) (cloudflowapplication.CloudflowStreamletDescriptorsDigestPair, string, error) {
```

Comment:
As I discussed with @agemooij in the past for now we should have this part in:
```
_, pullError := PullImage(cli, imageName)	
	if pullError != nil {	
		util.LogAndExit("Failed to pull image %s: %s", imageName, pullError.Error())	
	}	
```

Reply:
fixed @skonto ..